archive/h/haskell/README.md

```diff
@@ -8,6 +8,7 @@ Welcome to Sample Programs in Haskell!
 - [Fizz Buzz in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/349)
 - [Fibonacci in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/543)
 - [Reverse a String in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/559)
+- [Baklava in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/573)
```

Comment:
Actually! I need a quick fix. Can you create a new issue for the article like #31 and link it here?